modules/repository-azure/build.gradle

```diff
@@ -72,11 +73,11 @@ dependencies {
   api "org.reactivestreams:reactive-streams:${versions.reactiveStreams}"
 
   implementation project(":modules:transport-netty4")
+  implementation("org.slf4j:slf4j-api:${versions.slf4j}")
 
-  runtimeOnly("org.slf4j:slf4j-api:${versions.slf4j}")
+  runtimeOnly "com.azure:azure-storage-internal-avro:${versions.azureAvro}"
```

Comment:
The `com.azure.storage.blob` module requires `com.azure.storage.internal.avro`, so it must be present during runtime resolution, even if not used directly by our code.